### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rudranshsharma123**

### CODE REVIEW

**Review by gr2m — APPROVED** [review]
Changes look good, yay! We will merge your changes shortly, together with other pull requests that address the same issue. We also added the `hacktoberfest-accepted` label so your pull request counts already, in case you are participating.

File: README.md [hunk 1]
```diff
@@ -17,12 +17,13 @@ Besides the admin-specific endpoints, there are differences between `api.github.
 Browsers
 </th><td width=100%>
 
-Load `@octokit/plugin-enterprise-server` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
+Load `@octokit/plugin-enterprise-server` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+
```
Comment (gr2m) [inline README.md:21]: Can you remove this empty line? This is the reason why the tests are failing, we are checking all the files to follow certain formatting rules (using https://prettier.io/, you'll see that a lot in JavaScript project). And one rule is not to have two empty lines in Markdown (.md) files

Comment (rudranshsharma123) [reply README.md:21]: Yes I'd love to but how will I do that I cannot see any option and I am very new to GitHub

Comment (gr2m) [reply README.md:21]: no problem, happy to help! 

A simple way is to edit the file directly on github.com. In the "Files changed" tab, find the 3 dots on the top right, below the "review changes" dropdown

image

When you click "Edit file", then you get back to the editor mode, where you can make the change. Once you are done, press the green "Commit changes" button on the bottom of the page

Comment (rudranshsharma123) [reply README.md]: Hello, Thank you very much for your time and help. I made the change. Can you let me know if the change is fine now?

Comment (gr2m) [reply README.md]: It looks good now, thank you 👍🏼

**Comment by gr2m** [comment]
Congratulations 🥳 We hope you enjoyed the process of sending the pull requests, and maybe learned a thing or two? If anything was unclear, or if you have any suggestions on how we can improve the suggestions in the issue #123, please let us know! It will help future first time contributors!

If you are looking to make another pull requests, look out for issues with the help wanted label. They are not as straight forward as the first-timers-only issues, but hey, you no longer are a first-timers 🙂 You can find help wanted issues [here](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Aoctokit+label%3A%22help+wanted%22).
    
You can help us by starring this repository, as well as our primary repository at <https://github.com/octokit/rest.js/> ⭐️ Stars help others to discover our project.
    
If you are interested in Octokit and future opportunities to contribute, you can subscribe to our announcement discussion at <https://github.com/octokit/rest.js/discussions/620>. We will post an update soon and will point out issues you can contribute to.

**Pull request closed without merging**